### PR TITLE
Add macos11,12 arm64, win-11-enterprise, and fedora36

### PR DESCRIPTION
**Pull request by cthorn42**
(IMAGES-1344,IMAGES-1373,IMAGES-1297,IMAGES-1378) for the corresponding tickets.

### CODE REVIEW

**Comment by cthorn42** [comment]
Creating a draft first to get people's opinion on the naming convention we're using for the macos 11|12 m1 naming. In most other places we've gone with arm64, but that isn't how Apple has been marketing it. I'm leaning towards what i put in this PR just to keep it in tune with what we've done in the past.

File: lib/beaker-hostgenerator/data.rb [hunk 1]
```diff
@@ -1706,6 +1722,16 @@ def osinfo
           :vmpooler => {
             'template' => 'win-10-1809-x86_64'
           }
+        },
+        'windows11ent-64' => {
+          :general => {
+            'platform'           => 'windows-11ent-64',
+            'packaging_platform' => 'windows-2012-x64',
+            'ruby_arch' => 'x64'
```
Comment (ekohl) [inline lib/beaker-hostgenerator/data.rb:1730]: Alignment is inconsistent

File: lib/beaker-hostgenerator/data.rb [hunk 2]
```diff
@@ -884,6 +884,14 @@ def osinfo
             'template' => 'macos-112-x86_64'
           }
         },
+	'osx11-arm64' => {
```
Comment (ekohl) [inline lib/beaker-hostgenerator/data.rb:887]: indenting looks off

**Review by ekohl — COMMENTED** [review]
Rather formatting trivial points, otherwise I think this looks ok.

I do wonder if we can programatically replace osx -> macos in vmpooler for OS X >= 11 though. That would save some duplication.

**Comment by joshcooper** [comment]
I'm :+1: on the naming scheme you have, e.g. `'template' => 'macos-12-arm64'`

**Comment by joshcooper** [comment]
I'm :+1: on the naming scheme you have, e.g. `'template' => 'macos-12-arm64'`

**Pull request closed without merging**